atd-vzv/src/views/summary/PeopleByDemographics.js

```diff
@@ -113,7 +113,7 @@ const PeopleByDemographics = () => {
   const createChartLabels = () => yearsArray().map(year => `${year}`);
 
   // Tabulate fatalities by demographics in data
-  const getData = categoryValue =>
+  const getData = (categoryValue, getFullData) =>
```

Comment:
@mateoclarke shared [this article](https://www.samanthaming.com/tidbits/34-better-boolean-variable-names/) with me last year about naming boolean variables that could help out here. At first glance, I thought `getFullData` was a function passed as an argument.

Maybe `getFullData` could be renamed to `isWholeNumber` or something like that ? Definitely just my opinion and your comment on line 164 clarifies all of this.

Reply:
maybe `isRoundedNumber`, then when you call it below in 203 & 204, you'd need to flip those. You may also consider using a default parameter value.
https://codeburst.io/understanding-default-parameters-in-javascript-5f3c4dcfc838
https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Functions/Default_parameters

Reply:
Thanks, y'all! I renamed the boolean to `isWholeNumber`and the associated object to `wholeNumbers`, I think this makes it cleaner!